src/main/java/com/github/tomakehurst/wiremock/junit/Stubbing.java

```diff
@@ -38,6 +38,8 @@ public interface Stubbing {
 
   void removeStub(StubMapping mappingBuilder);
 
+  void removeStub(UUID id);
```

Comment:
Without a default implementation, this is a binary compatibility breaking change.
You can add a default implementation by chaining `getSingleStubMapping(id)` and the existing removal method.

P.S: no strong opinion, but maybe the method should be actually returning the removed stubbing or `null` if it does not exist

Reply:
Not worried about binary compatibility because it should only affect implementers of the `Stubbing` interface, and I can't think of a reason for anyone else to do that.

Agree returning the deleted stub or null would be a better interface design (albeit not consistent with the existing one), so let's do that.